src/devices/infactory.c

```diff
@@ -1,84 +1,127 @@
 /** @file
     inFactory outdoor temperature and humidity sensor.
+    Other brand names: nor-tec, GreenBlue, DAY
```

Comment:
Please add the info in the decoder comment below, not in the copyright.

Reply:
OK.